src/web/app/components/comment-box/comment-table-modal/comment-table-modal.component.spec.ts

```diff
@@ -52,4 +54,68 @@ describe('CommentTableModalComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('should set isAddingNewComment to true in the model', () => {
+    const ngOnChangesSpy: SpyInstance = jest.spyOn(component.modelChange, 'emit');
+    const testModel: CommentTableModel = {
+      commentRows: [],
+      newCommentRow: {
+        commentEditFormModel: {
+          commentText: '',
+          isUsingCustomVisibilities: false,
+          showCommentTo: [],
+          showGiverNameTo: [],
+        },
+        isEditing: true,
+      },
+      isAddingNewComment: false,
+      isReadOnly: false,
+    };
+    component.model = testModel;
+    component.ngOnChanges();
+    expect(ngOnChangesSpy).toHaveBeenCalledWith({
+      ...testModel,
+      isAddingNewComment: true,
+    });
+  });
+
+  it('should trigger an event to delete comment from comments table', () => {
```

Comment:
```suggestion
  it('should emit a DeleteCommentEvent with the correct index when triggerDeleteCommentEvent is called', () => {
```

let's follow this format when describing tests for event emissions, do the same for the other tests that emit events